src/containers/stats/courttype/EnrollmentsAndStatusByCourtType.js

```diff
@@ -193,19 +195,25 @@ const EnrollmentsAndStatusByCourtType = ({
     const courtType :string = v.y;
     if (!isDefined(courtType)) return [];
     const active = activeEnrollmentsByCourtType.get(courtType, 0);
+    const becameActive = becameActiveEnrollmentsByCourtType.get(courtType, 0);
     const closed = closedEnrollmentsByCourtType.get(courtType, 0);
     const jobSearch = jobSearchEnrollmentsByCourtType.get(courtType, 0);
     const successful = successfulEnrollmentsByCourtType.get(courtType, 0);
     const unsuccessful = unsuccessfulEnrollmentsByCourtType.get(courtType, 0);
-    return [
+    const statusCounts = [
       { title: courtType, value: '' },
       { title: 'successful', value: successful },
       { title: 'unsuccessful', value: unsuccessful },
       { title: 'closed', value: closed },
       { title: 'active', value: active },
+      { title: '[became active]', value: becameActive },
```

Comment:
why does this have brackets?

Reply:
i wanted to differentiate it somehow from the other categories, because the other categories add up to the total... but if you have any other ideas, i'm very open to changing this